packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -101,12 +117,28 @@ export function ListViewItem(props) {
         ref={cellRef}
         {...mergedProps}>
         <Grid UNSAFE_className={listStyles['react-spectrum-ListViewItem-grid']}>
-          {showCheckbox && (
+          {isListDraggable &&
+            <div className={listStyles['react-spectrum-ListViewItem-draghandle-container']}>
+              {showDragHandle &&
+                <FocusRing focusRingClass={classNames(listStyles, 'focus-ring')}>
+                  <div
+                    {...buttonProps as React.HTMLAttributes<HTMLElement>}
+                    className={listStyles['react-spectrum-ListViewItem-draghandle-button']}
+                    ref={dragButtonRef}
+                    data-testid="draghandle"
+                    draggable="true">
+                    <DragHandle />
+                  </div>
+                </FocusRing>
+              }
```

Comment:
A bit of weirdness when you perform a drag and drop operation via keyboard: the focus ring will remain on the drag handle and the up/down/left/right arrow won't move your through the rows/row items. Tabbing brings you out of the list and you can tab back into the list but the original focus ring on the drag handler doesn't disappear

Reply:
Might be something with the drag hooks themselves?